activation/post.go

```diff
@@ -89,6 +89,11 @@ func (c *PostClient) Execute(challenge []byte) (*types.PostProof, error) {
 }
 
 func (c *PostClient) Reset() error {
+	ok, err := c.IsInitialized()
+	if !ok || err != nil {
+		return fmt.Errorf("post not initialized, cannot reset it")
+	}
```

Comment:
I don't see where you reset it here ?

Reply:
if we're not initialized, there is nothing to reset

Reply:
line 100